### ISSUE TITLE
Get Unhandled exception in TaskEditDialog for v1 if try to change "configure for"

### ISSUE THREAD

**Issue by dahall** [dahall/TaskScheduler#188]
Hello, 

If I set forceV1=true on V2 systems and try to change a task via TaskEditDialog I catch an exception.
There is ability to change version via dialog and the exception occurs if you try to change it. 
(Dialog disable this option for V1 system)

```
Exception: Microsoft.Win32.TaskScheduler.NotV1SupportedException
Message: TaskSettings.set_Compatibility is not supported on Task Scheduler 1.0
Stack Trace:
   at Microsoft.Win32.TaskScheduler.TaskSettings.set_Compatibility(TaskCompatibility value)
   at Microsoft.Win32.TaskScheduler.TaskPropertiesControl.taskVersionCombo_SelectedIndexChanged(Object sender, EventArgs e)
   at System.Windows.Forms.ComboBox.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ComboBox.WmReflectCommand(Message& m)
   at System.Windows.Forms.ComboBox.WndProc(Message& m)
   at System.Windows.Forms.DisabledItemComboBox.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

Originally posted: 2016-02-01T05:07:31

**Comment by dahall** [comment]
Thank you. This is fixed in 2.5.13 due shortly.

Originally posted: 2016-02-06T08:06:36

**Comment by dahall** [comment]
Originally posted: 2016-02-07T10:50:16
